packages/nextjs-experimental/next.config.js

```diff
@@ -9,4 +9,16 @@ const nextConfig = {
       },
     ];
   },
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: 'oaidalleapiprodscus.blob.core.windows.net',
+        port: '',
+        pathname: '**',
+      },
+    ],
+  },
 };
+
+module.exports = nextConfig;
```

Comment:
Turns out we weren't exporting this.
I did notice an issue though: after this fix, the `yarn workspace docs start` command loads the `localhost:3000/basic-completion` page by default which is not found. Not sure how they are related or how to fix TBH.

Reply:
I think I only notice that when I have the docs and nextjs servers running at the same time (which it generally seems like I shouldn't be doing. :smile:)